src/ng/sce.js

```diff
@@ -855,11 +921,9 @@ function $SceProvider() {
      * Shorthand method.  `$sce.trustAsUrl(value)` →
      *     {@link ng.$sceDelegate#trustAs `$sceDelegate.trustAs($sce.URL, value)`}
      *
-     * @param {*} value The value to trustAs.
-     * @returns {*} An object that can be passed to {@link ng.$sce#getTrustedUrl
-     *     $sce.getTrustedUrl(value)} to obtain the original value.  (privileged directives
-     *     only accept expressions that are either literal constants or are the
-     *     return value of {@link ng.$sce#trustAs $sce.trustAs}.)
+     * @param {*} value The value to mark as trusted for URL context.
+     * @returns {*} A wrapped version of value that can be used as a trusted variant
+     *   of your `value` in URL context (currently unused.)
```

Comment:
(currently untrusted) is a bit misleading, maybe `used as a trusted variant of your previously untrusted value`?